CinebaseTests/Use Cases/Movies/NowPlaying/NowPlayingPresenterSpec.swift

```diff
@@ -28,7 +28,7 @@ class NowPlayingPresenterSpec: QuickSpec {
                 mockView = MockNowPlayingView()
                 mockRouter = MockRouter()
                 mockAPIClient = MockMoviesAPIClient()
-                sut = NowPlayingPresenter(view: mockView, moviesAPIClient: mockAPIClient)
+sut = NowPlayingPresenter(view: mockView, moviesAPIClient: mockAPIClient)
```

Comment:
i dont change it, so i dont understand why the git show the difference, in xcode looks the same indentation